.github/workflows/test.yaml

```diff
@@ -368,3 +398,14 @@ jobs:
       - name: Run make verify-helm-charts
         run: |
           make verify-helm-charts
+  required-checks:
+    name: CI Test Required Checks
+    # This check adds a list of checks to one job to simplify adding settings to the repo.
+    # If a new check is added in this file, and it should be retested on entry to the merge queue,
+    # it needs to be added to the list below aka needs: [ existing check 1, existing check 2, new check ].
+    needs: [ unit-tests, controllers-integration-tests, bare-k8s-integration-tests, gatewayapi-integration-tests, gatewayapi-provider-integration-tests, verify-manifests, verify-bundle, verify-fmt, test-scripts, verify-generate, verify-go-mod, verify-helm-charts ]
```

Comment:
This is a big list. I need someone more familiar with the code base to say whether all of these checks are required to be retested once the pr is added to the merge-queue

Reply:
This list should just be all the jobs in this workflow for now, that will probably be the case for most if not all of them.